### PR TITLE
net-im/qtox: Added qtox version 1.17.3

### PR DESCRIPTION
**Pull request by rollagnier**
Package-Manager: Portage-3.0.13, Repoman-3.0.2

### CODE REVIEW

**Review by juippis — COMMENTED** [review]
Thanks, looks good! But we still require the `Signed-off-by:` line with your real name present, 
https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin

Check any commit to ::gentoo as an example, 
7ccdbea27457da611c7aa7dfb9008d087b1aef77 (randomly picked newest commit). It has two signed-off-by lines, one from the author and one from the committer. We need **yours** before this can be merged.

Now you can *amend* your commit, or just comment the signed-off-by line here in Github, and I'll add it to you while merging.

In future, you can automate adding the signed-off-by line by adding `SIGNED_OFF_BY="My Name <my@e.mail>" in your *make.conf* while using repoman to commit, or by using *pkgdev commit* too.

**Review by rollagnier — COMMENTED** [review]
Signed-off-by: Rémi Ollagnier <remi.ollagnier@free.fr>

File: net-im/qtox/qtox-1.17.3.ebuild
Comment (juippis) [inline net-im/qtox/qtox-1.17.3.ebuild:11]: Had to update this to point out to proper release download link, 
https://github.com/qTox/qTox/releases/tag/v1.17.3

Other than that, works perfectly, thanks!

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**
